x-pack/plugins/uptime/server/lib/requests/search/monitor_summary_iterator.ts

```diff
@@ -117,17 +149,20 @@ export class MonitorGroupIterator {
       this.bufferPos = 0;
     }
 
-    const results = await this.chunkFetcher(this.queryContext, this.searchAfter, size);
+    const results = await this.chunkFetcher(this.queryContext, this.searchAfter, CHUNK_SIZE);
     // If we've hit the end of the stream searchAfter will be empty
-
-    results.monitorGroups.forEach((mig: MonitorGroups) => this.buffer.push(mig));
+    results.monitorSummaries.forEach((ms: MonitorSummary) => this.buffer.push(ms));
     if (results.searchAfter) {
       this.searchAfter = results.searchAfter;
     }
 
+    if (!results.searchAfter) {
+      this.endOfResults = true;
+    }
+
     return {
-      gotHit: results.monitorGroups.length > 0,
-      hasMore: !!results.searchAfter,
+      gotHit: results.monitorSummaries.length > 0,
+      hasMore: this.endOfResults,
```

Comment:
i don't see hasMore being used anywhere, should be removed.

Reply:
Good catch, will do